app/boards/shields/kyria/keymaps/default/keymap.overlay

```diff
@@ -0,0 +1,50 @@
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/keymap-transform.h>
+#include <keymap.h>
+
+/ {
+	chosen {
+		zmk,keymap = &keymap0;
+	};
+	
+	keymap0: keymap {
+		compatible = "zmk,keymap";
+		label ="Default Kyria Keymap";
+		layers = <&default>;
+		transform = <&default_transform>;
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,keymap-transform";
+		map = <
+			IDENTITY_ROW(0, 6)       OFFSET_ROW(6, 0, 6)
```

Comment:
This uses some higher level wrappers to make this quick. I will tweak the include file a bit so we can also have per item defines here, e.g. `<POS(1, 2) POS(2,1) ... >` so you can manually define the mapping for each entry to the matrix row/column explicitly.

Reply:
Updated the naming here, and made a `KT_ROW_COL` macro as well, to let you handcode *all* the mappings, if the `KT_ID_ROW` and `KT_OFFSET_ROW` aren't usable when doing a keyboard wit a funky matrix layout.

Reply:
Ok, so completely removed these "helper" macros just now, in favor of smaller, per-entry macros for each individual location, for clarity on the use here. Marking this one resolved.